api/standard/clause_8_ogcapi_processes.adoc

```diff
@@ -1,18 +1,18 @@
 [[processes-profile]]
-== Routing API as a profile of OGC API Processes
+== Routing API as a profile of _OGC API - Processes_
```

Comment:
As I was replacing some of these WPS, I tried to be consistent across. @ghobona had previously clarified that the proper style as OGC API - {Name}. Is this alternate style also OK for referencing specifications as opposed to the document?

I personally prefer things to be fully consistent, but it would be good to get some guidance on the proper approach in that case of referencing the specifications rather than the document.

Reply:
I have not updated these, I think we can harmonize this later.